doc/sphinx/source/fota.txt

```diff
@@ -0,0 +1,33 @@
+######################
+FOTA software of MuxPi
+######################
+
+This manual presents the main functionalities of fota.
+
+The MuxPi together with NanoPi Neo have two microSD card slots. The slot on the NanoPi Neo is used to boot up the MuxPi and install software such as fota or stm. When using FOTA, the card slot on the muxPi must be used.
+
+Find the correct device by running ``lsblk``
+
+**********
+Fota usage
+**********
+
+Overall, the idea of FOTA is to flash images to drives. FOTA has the same purpose as ``dd if="rootfs.img" of="/dev/sdX"``. 
```

Comment:
nope, it is similar to dd but it takes tarball over http as input, untars it and flashes on the fly.  there would be no reason for developing this tool if it would be the same as ``dd``